Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -20,50 +25,43 @@ implemented the Hook Concept itself doesn't prevent it from being used in any wa
 Using hooks
 ===========
 
-The two lines of code below are an example of how a hook is used for
+The two lines of code below are an example of how a hook can be used for
 clear-cache post-processing. The objective of this could be to perform
 additional actions whenever the cache is cleared for a specific page:
 
-.. code-block:: php
-   :caption: EXT:site_package/ext_localconf.php
-
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_tcemain.php']['clearCachePostProc'][] =
-      \MyVendor\Package\Hook\DataHandlerHook::class . '->postProcessClearCache';
+..  literalinclude:: _ext_localconf_addhook.php
+    :language: php
```

Comment:
language is not really necessary as it is the default. But is does not hurt.

Reply:
Just like with the code-blocks we should always use a language. This way it is more transparent how to make snippets for other languages. Also we decided not to rely on defaults.